source/org/zfin/publication/repository/HibernatePublicationRepository.java

```diff
@@ -83,148 +83,48 @@ public class HibernatePublicationRepository extends PaginationUtil implements Pu
 
     private ProfileRepository profileRepository = RepositoryFactory.getProfileRepository();
 
-    public int getNumberOfPublications(String abstractText) {
-
-        Criteria query = HibernateUtil.currentSession().createCriteria(Publication.class);
-        query.add(Restrictions.ilike("abstractText", abstractText, MatchMode.ANYWHERE));
-        query.setProjection(Projections.count("abstractText"));
-        return ((Number) query.uniqueResult()).intValue();
-    }
-
-    /**
-     * Retrieve all distinct publications that contain a high quality probe
-     * with a rating of 4.
-     *
-     * @param anatomyTerm Anatomy Term
-     * @return list of publications
-     */
-    public List<Publication> getHighQualityProbePublications(GenericTerm anatomyTerm) {
-        Session session = HibernateUtil.currentSession();
-        String hql = "SELECT distinct exp.publication FROM ExpressionExperiment exp, ExpressionResult res, Clone clone   " +
-            "WHERE res.entity.superterm = :aoTerm " +
-            "AND res.expressionExperiment = exp " +
-            "AND exp.probe = clone " +
-            "AND clone.rating = 4 " +
-            "AND clone.problem is null";
-        Query query = session.createQuery(hql);
-        query.setParameter("aoTerm", anatomyTerm);
-        List<Publication> list = query.list();
-        return list;
-    }
-
     public List<Publication> getExpressedGenePublications(String geneID, String anatomyItemID) {
         Session session = HibernateUtil.currentSession();
-        String hql = "SELECT distinct publication FROM Publication publication, ExpressionExperiment exp, ExpressionResult res   " +
-            "WHERE (res.entity.superterm.zdbID = :aoZdbID OR" +
-            "       res.entity.subterm.zdbID = :aoZdbID) " +
-            "AND publication = exp.publication " +
-            "AND res.expressionExperiment = exp " +
-            "AND exp.gene.zdbID = :zdbID " +
-            "AND res.expressionFound = :expressionFound ";
+        String hql = """
+            SELECT distinct publication FROM Publication publication, ExpressionExperiment exp, ExpressionResult res  \s

```

Comment:
What is the \s character at the end of this line?

Reply:
I think it was added automatically by the intellij conversion to text block. It must be to preserve white space at the end of a line. It was unnecessary, so I removed it.